frontend/src/scenes/notebooks/Nodes/NotebookNodeCohort.tsx

```diff
@@ -0,0 +1,180 @@
+import { createPostHogWidgetNode } from 'scenes/notebooks/Nodes/NodeWrapper'
+import { NotebookNodeType, PropertyFilterType } from '~/types'
+import { useActions, useValues } from 'kea'
+import { urls } from 'scenes/urls'
+import { LemonSkeleton } from 'lib/lemon-ui/LemonSkeleton'
+import { notebookNodeLogic } from './notebookNodeLogic'
+import { NotebookNodeProps } from '../Notebook/utils'
+import { useEffect, useMemo } from 'react'
+import clsx from 'clsx'
+import { NotFound } from 'lib/components/NotFound'
+import { cohortEditLogic } from 'scenes/cohorts/cohortEditLogic'
+import { IconCohort, IconPerson, InsightsTrendsIcon } from 'lib/lemon-ui/icons'
+import { Query } from '~/queries/Query/Query'
+import { LemonDivider, LemonTag } from '@posthog/lemon-ui'
+import { DataTableNode, NodeKind } from '~/queries/schema'
+
+const Component = ({ attributes, updateAttributes }: NotebookNodeProps<NotebookNodeCohortAttributes>): JSX.Element => {
+    const { id } = attributes
+
+    const { expanded } = useValues(notebookNodeLogic)
+    const { setExpanded, setActions, insertAfter } = useActions(notebookNodeLogic)
+
+    const { cohort, cohortLoading, cohortMissing, query } = useValues(cohortEditLogic({ id }))
+    const { setQuery } = useActions(cohortEditLogic({ id }))
+
+    const title = cohort ? `Cohort: ${cohort.name}` : 'Cohort'
+
+    const modifiedQuery = useMemo<DataTableNode>(() => {
+        return {
+            ...query,
+            embedded: true,
+            // TODO: Add back in controls in a way that actually works...
+            full: false,
+            showElapsedTime: false,
+            showTimings: false,
```

Comment:
We should hide the editor button here too

```suggestion
            showTimings: false,
            showOpenEditorButton: false,
```